lib/YuiRestClient/Http/WidgetController.pm

```diff
@@ -50,11 +49,17 @@ sub find {
         params => $args
     );
 
-    YuiRestClient::Logger->get_instance()->debug('Finding widget by url: ' . $uri);
+    YuiRestClient::Logger->get_instance()->debug("Finding widget by url: $uri");
```

Comment:
why is it that use YuiRestClient::Logger;` is not required any more?

Reply:
I removed it by mistake. The question now, how it worked without the import:)

Reply:
I guess non of those `use`s are needed because there are in the same namespace, but it is more clear that way.